WcaOnRails/app/views/admin/compute_auxiliary_data.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide(:title, 'Compute Auxiliary Data') %>
+
+<div class="container">
+  <%= render layout: "nav" do %>
+    <h2><%= yield(:title) %></h2>
+    <% if @auxiliary_data_start.date.nil? %>
```

Comment:
With my subclass suggestion above, this would become `ComputeAuxiliaryData.start_timestamp.date.ni?` instead.